modules/ebi-metagenomics/proovframe/tests/main.nf.test

```diff
@@ -0,0 +1,33 @@
+nextflow_process {
+
+    name "Test Process PROOVFRAME"
+    script "../main.nf"
+    process "PROOVFRAME"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "proovframe"
+
+    test("mytest1 - faa_fasta") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file("${moduleDir}/tests/fixtures/emales.faa", checkIfExists: true),
+                    file("${moduleDir}/tests/fixtures/emales-reads-sim.fa", checkIfExists: true),
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.versions).match() }  // Adding the version assertion
```

Comment:
Should test all outputs, not just versions:
{ assert snapshot(process.out).match() }